modules/ducktests/tests/ignitetest/services/utils/ignite_spec.py

```diff
@@ -147,6 +146,8 @@ def __init__(self, modules, **kwargs):
         libs.append("log4j")
         libs = list(map(lambda m: self.path.module(m) + "/*", libs))
 
+        libs.append(IgnitePath(DEV_BRANCH).module("ducktests") + "/*")
```

Comment:
Why do you add ducktests to NodeSpec?

Reply:
At this test, we have a special node filter (see CellularAffinityBackupFilter.java) and we have to deploy it to all server node's classpath.

Reply:
There is a code duplication - the same in ApplicationSpec. Could you please move this code to function?

Reply:
Seems, we have to plan some refactoring here (at specs), can we postpone this to refactoring?